README.md

```diff
@@ -99,11 +101,11 @@ The editors will now review and edit the post. Please respond to any questions t
 
 These steps are completed by the editors of the blog. As editor, you might ask questions or make suggestions to the author of the post. You might also make edits directly in the post to prepare it for publishing.
 
-1. Review the post on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/) as linked from the issue.
+1. Review the post on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/) as linked from the issue.
 
    Ask the author to make changes by adding review comments to the PR.
 
-   For edits such as punctuation, formatting, highlighting, adding SEO details, or larger changes discussed with the author, the editor can make the edits directly in the author's branch and push the changes to `draft` branch, then rebuild the [draft site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) to check them.
+   For edits such as punctuation, formatting, highlighting, adding SEO details, or larger changes discussed with the author, the editor can make the edits directly in the author's branch and push the changes to `draft` branch, which will automatically rebuiild [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) and [draft site](https://draft-openlibertyio.mybluemix.net/blog/) where you can verify the changes.
```

Comment:
which will automatically rebuiild [blog-draft site
-->
which automatically rebuilds the [blog-draft site